index.html

```diff
@@ -11,169 +11,223 @@
     
   </head>
   <body>
-    <div class="maincont">
-
+    <div class="outer-maincont">
       <header class="app-bar">
-        <nav class="lower-bar">
-          <a href="#" class="logoo">Somarib Emmanuel</a> 
-          <div class="menu-icons"><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></div>
-        </nav>
+
+          <nav class="lower-bar">
+              <ul><li><a href="#" class="logoo">Somarib Emmanuel</a></li></ul>
+              <ul class="big-menu">
+                  <li><a href="">Portfolio</a></li>
+                  <li><a href="">About</a></li>
+                  <li><a href="">Contact</a></li>
+                  <li><a href=""><img class="" src="image/bg-envlope.png" alt="menu icon"></a></li>
+              </ul>
+              <ul class="menu-icons"><li><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li></ul>
+          </nav>
       </header>
+      <main class="main-cont">
+          <article class="header-section">
+              <div class="header-inner-cont">
+                  <section class="inner-section">
+                      <h1 class="myname">Somarib Emmanuel</h1>
```

Comment:
Kindly fix the font of this heading according to the Figma design 😀

Your design:

![image](https://user-images.githubusercontent.com/73714615/187038367-a11ad65b-4771-4968-9b93-cbf1e26d1272.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187038374-bba73060-d196-491c-894a-e099c79f3ade.png)